### ISSUE TITLE
CNVKIT_REFERENCE error

### ISSUE THREAD

**Issue by paolo-kunderfranco** [nf-core/sarek#951]
### Description of the bug

Dear All,
I am running sarek v 3.0.2, command as pasted below.
Seems like is not finding the referene genome.
Could you please assist me? 
Best

here is my output

```
-[nf-core/sarek] Pipeline completed with errors-
Error executing process > 'NFCORE_SAREK:SAREK:PREPARE_CNVKIT_REFERENCE:CNVKIT_REFERENCE (Homo_sapiens_assembly38.fasta)'

Caused by:
  Process `NFCORE_SAREK:SAREK:PREPARE_CNVKIT_REFERENCE:CNVKIT_REFERENCE (Homo_sapiens_assembly38.fasta)` terminated with an error exit status (1)

Command executed:

  cnvkit.py \
      reference \
      --fasta Homo_sapiens_assembly38.fasta \
      --targets Twist_Exome_RefSeq_targets_hg38.bed \
      --antitargets home.antitarget.bed \
      --output cnvkit.reference.cnn \
  
  
  cat <<-END_VERSIONS > versions.yml
  "NFCORE_SAREK:SAREK:PREPARE_CNVKIT_REFERENCE:CNVKIT_REFERENCE":
      cnvkit: $(cnvkit.py version | sed -e "s/cnvkit v//g")
  END_VERSIONS

Command exit status:
  1

Command output:
  (empty)

Command error:
  Detected file format: bed
  Detected file format: bed
  Calculating GC and RepeatMasker content in Homo_sapiens_assembly38.fasta ...
  Traceback (most recent call last):
    File "/usr/local/lib/python3.9/site-packages/pyfaidx/__init__.py", line 369, in __init__
      self.file = self._fasta_opener(filename, 'r+b'
  FileNotFoundError: [Errno 2] No such file or directory: 'Homo_sapiens_assembly38.fasta'
  
  During handling of the above exception, another exception occurred:
  
  Traceback (most recent call last):
    File "/usr/local/bin/cnvkit.py", line 9, in <module>
      args.func(args)
    File "/usr/local/lib/python3.9/site-packages/cnvlib/commands.py", line 520, in _cmd_reference
      ref_probes = reference.do_reference_flat(args.targets, args.antitargets,
    File "/usr/local/lib/python3.9/site-packages/cnvlib/reference.py", line 30, in do_reference_flat
      gc, rmask = get_fasta_stats(ref_probes, fa_fname)
    File "/usr/local/lib/python3.9/site-packages/cnvlib/reference.py", line 461, in get_fasta_stats
      gc_rm_vals = [calculate_gc_lo(subseq)
    File "/usr/local/lib/python3.9/site-packages/cnvlib/reference.py", line 461, in <listcomp>
      gc_rm_vals = [calculate_gc_lo(subseq)
    File "/usr/local/lib/python3.9/site-packages/cnvlib/reference.py", line 487, in fasta_extract_regions
      with pyfaidx.Fasta(fa_fname, as_raw=True) as fa_file:
    File "/usr/local/lib/python3.9/site-packages/pyfaidx/__init__.py", line 996, in __init__
      self.faidx = Faidx(
    File "/usr/local/lib/python3.9/site-packages/pyfaidx/__init__.py", line 378, in __init__
      raise FastaNotFoundError(
  pyfaidx.FastaNotFoundError: Cannot read FASTA file Homo_sapiens_assembly38.fasta
```

### Command used and terminal output

```console
nextflow run nf-core/sarek --input $wd/samplesheet.csv --outdir $wd/outdir
--genome GATK.GRCh38 --only_paired_variant_calling TRUE \
--wes TRUE \
--intervals $panel/Twist_Exome_RefSeq_targets_hg38.bed \
--tools strelka,mutect2,msisensorpro,cnvkit,manta,snpeff,vep,merge -profile singularity \
--max_cpus $SLURM_NTASKS --save_output_as_bam TRUE \
-c $wd/nextflow.config -r 3.0.2
```

### System information

slurm
module load nextflow/22.10.4
module load graphviz/2.50.0
module load singularity/3.4.2

**Comment by FriederikeHanssen** [comment]
Hi! Apologies, just saw this today. (If noone replies, also feel free to ping us on slack in the #sarek channel, you can join here: nf-co.re/join ) 
Not sure why it can't find the reference genome. I assume, mapping and so on ran successfully?

**Comment by FriederikeHanssen** [comment]
This looks rather similar to #920. but for both I don't see what the issue could be

**Comment by FriederikeHanssen** [comment]
@ameynert fixed #920, since the issues look to be similar I am closing this for now. If you experience this again, please re-open :)
